File: storydoc-backend/src/main/java/io/storydoc/server/code/app/StitchConfigDTO.java
package io.storydoc.server.code.app;

import io.storydoc.server.code.domain.StitchConfigCoordinate;
import io.storydoc.server.code.domain.StitchConfigId;
import io.storydoc.server.storydoc.app.dto.StoryDocSummaryDTO;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;

@Data
@AllArgsConstructor
@Builder
public class StitchConfigDTO {
    private StitchConfigCoordinate stitchConfigCoordinate;
    private String dir;
    private String name;
    private StoryDocSummaryDTO storyDocSummary;

}

File: storydoc-frontend/src/app/api/models/stitch-config-dto.ts
/* tslint:disable */
/* eslint-disable */
import { StitchConfigCoordinate } from './stitch-config-coordinate';
import { StoryDocSummaryDto } from './story-doc-summary-dto';
export interface StitchConfigDto {
  dir?: string;
  name?: string;
  stitchConfigCoordinate?: StitchConfigCoordinate;
  storyDocSummary?: StoryDocSummaryDto;
}
